### DISCUSSION TITLE
file not found... but it is there.

### DISCUSSION THREAD

**Discussion by MrGingerSir222** [JonathanFly/bark#21] [Q&A]
anyone else getting this kind of error once you try running the test script? I checked, and the file it says isn't there is indeed in that exact location with that exact name, so I'm at a loss:

"FileNotFoundError: Could not find module 'C:\Users\user\AppData\Local\Programs\Python\Python310\Lib\site-packages\torchaudio\lib\libtorchaudio.pyd' (or one of its dependencies). Try using the full path with constructor syntax."

**Comment by JonathanFly** [comment]
This is probably fixed in the new version?
